crypto/evp/p_lib.c

```diff
@@ -1976,16 +1982,10 @@ int evp_pkey_downgrade(EVP_PKEY *pk)
         evp_pkey_free_it(&tmp_copy);
         rv = 1;
     } else {
-        /* Grab the temporary lock to avoid lock leak */
-        tmp_lock = pk->lock;
-
         /* Restore the original key */
         *pk = tmp_copy;          /* |pk| now owns THE lock */
```

Comment:
`/* |pk| now owns THE lock */` isn't fitting any more, is it?

Reply:
I've removed this now, thanks